README.md

```diff
@@ -24,13 +24,15 @@ $ git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator
 Build the operator image
 
 ```bash
-make image IMAGE=<my repo>:<my tag>
+IMAGE_TAG=<my repo>:<my tag>
```

Comment:
Or should we encourage customizing only `IMAGE_REPO` instead and let image tag name come from Makefile, WDYT?
```
IMAGE_REPO=<my-repo> make image
```

In this case I would not change the initialization of `IMAGE_TAG` in Makefile

Reply:
sounds good to me. is just that when I am testing new things I use personal tags

example: `quay.io/eduardoarango/nfd:fixsomething`

I am ok with both options.

Reply:
> sounds good to me.

My thinking behind this was that `IMAGE_TAG_NAME` and `IMAGE_REGISTRY` are also used independently (at least on the operand side). Thus overriding `IMAGE_TAG` does not always have the desired effect as we do not reverse-parse `IMAGE_REGISTRY` and `IMAGE_TAG_NAME` from that. Thus, prefer not to have `IMAGE_TAG` as the one of the customization variables advertised to users.


> is just that when I am testing new things I use personal tags
> 
> example: `quay.io/eduardoarango/nfd:fixsomething`

You can still do that via explicit makefile variable overrides (implies you know what you're doing :smile: ) like
```
make image IMAGE_TAG=quay.io/eduardoarango/nfd:fixsomething
```

Does this make sense?

Reply:
makes total sense :)

Reply:
K. Would you then change the PR so that only README is changed. Like this:
```
IMAGE_REGISTRY=<my registry> make image 
```